myapp/app/views/layouts/_tasks_flash.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if flash[:success].present? %>
+  <div class="alert alert-success" role="alert"><%= flash[:success] %></div>
+<% end %>
+<% if flash[:danger].present? %>
+  <div class="alert alert-danger" role="alert"><%= flash[:danger] %></div>
+<% end %>
```

Comment:
Add a new line at the end of the file. Its Rakuma's coding practice

Reply:
Thank you.
I'll make sure to add a new line at the end of the file!